osu.Game/Online/Chat/NowPlayingCommand.cs

```diff
@@ -59,9 +67,31 @@ protected override void LoadComplete()
                     break;
             }
 
-            string beatmapString = beatmapInfo.OnlineID > 0 ? $"[{api.WebsiteRootUrl}/b/{beatmapInfo.OnlineID} {beatmapInfo}]" : beatmapInfo.ToString();
+            string beatmapString()
```

Comment:
Method names are generally expected to be verb phrases rather than noun phrases (i.e. `getBeatmapPart()` or something).